packages/accounts-base/accounts_common.js

```diff
@@ -184,3 +184,30 @@ Accounts._tokenExpiresSoon = function (when) {
     minLifetimeMs = minLifetimeCapMs;
   return new Date() > (new Date(when) - minLifetimeMs);
 };
+
+// Callback exceptions are printed with Meteor._debug and ignored.
+onLoginHook = new Hook({
+  debugPrintExceptions: "onLogin callback"
+});
+onLoginFailureHook = new Hook({
+  debugPrintExceptions: "onLoginFailure callback"
+});
+
+
+/**
+ * @summary Register a callback to be called after a login attempt succeeds.
+ * @locus Anywhere
```

Comment:
To get this change to stick, run `./scripts/admin/jsdoc/jsdoc.sh` from your Meteor checkout, then commit the new `data.js` file.